dotCMS/src/main/java/com/dotcms/contenttype/business/FieldFactory.java

```diff
@@ -12,7 +12,7 @@ default FieldFactory instance() {
         return new FieldFactoryImpl();
     }
 
-
+    public final static String GENERIC_FIELD_VAR="field";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)